components/Brands/index.tsx

```diff
@@ -4,21 +4,7 @@ import SingleBrand from "./SingleBrand";
 import brandData from "./brandData";
 
 const Brands = () => {
-  return (
-    <>
-      {/* <!-- ===== Clients Start ===== --> */}
-      <section className="border border-x-0 border-y-stroke bg-alabaster py-11 dark:border-y-strokedark dark:bg-black">
-        <div className="mx-auto max-w-c-1390 px-4 md:px-8 2xl:px-0">
-          <div className="grid grid-cols-3 items-center justify-center gap-7.5 md:grid-cols-6 lg:gap-12.5 xl:gap-29">
-            {brandData.map((brand, key) => (
-              <SingleBrand brand={brand} key={key} />
-            ))}
-          </div>
-        </div>
-      </section>
-      {/* <!-- ===== Clients End ===== --> */}
-    </>
-  );
+
 };
```

Comment:
You won't have to remove component's part. you just can comment or remove the component from the app/pages file
![image](https://github.com/user-attachments/assets/5f588d2f-615e-4a42-8ecd-4a82d649175b)